web/src/features/panels/zone/ShareButton.tsx

```diff
@@ -0,0 +1,97 @@
+import { Button, ButtonProps } from 'components/Button';
+import { Toast, useToastReference } from 'components/Toast';
+import { isiOS, isMobile } from 'features/weather-layers/wind-layer/util';
```

Comment:
Would be nice to try not use the windlayer stuff here if possible. 

For mobile I'd love to unify all the checks but now isn't the time so I think this is the most up to date one `useIsMobile`.

For iOS check we kinda have one in here isOldIOSVersion that we could extract into a util that isn't in the wind layer.

Reply:
Agreed, it's definitely time to extract this